pythonz/installer/pythoninstaller.py

```diff
@@ -144,8 +144,6 @@ def __init__(self, version, options):
                 raise Exception
             if options.framework:
                 self.configure_options.append('--enable-framework=%s' % os.path.join(self.install_dir, 'Frameworks'))
-            if options.shared:
-                self.configure_options.append('--enable-shared')
```

Comment:
Sorry, _now_ I remember. I was trying to fix building Stackless on macOS. I don't think this change fixes it, though. I'll add these back.